src/s2wasm.h

```diff
@@ -1193,6 +1198,42 @@ class S2WasmBuilder {
         }
       }
     }
+    if (!!startFunction) {
```

Comment:
isn't startFunction null if not used? Then checking is() which checks the pointer is not null would be fine?

or are you saying startFunction has a valid pointer that points to an empty string?